### ISSUE TITLE
TFS build task incorrectly setting updateAssemblyInfoFlag

### ISSUE THREAD

**Issue by roryprimrose** [GitTools/GitVersion#855]
I have not checked the "Update AssemblyInfo files" option, yet the build log contains the following:

```
Invoking GitVersion with C:\a\1\s /output buildserver /nofetch /updateassemblyinfo true
```

From what I can tell in the powershell, the /updatessemblyinfo true should not be added to the command line. The code itself looks fine so I don't know what is happening there.

**Comment by pascalberger** [comment]
This was fixed recently with #847, and I don't think it is included in the recent release.

@JakeGinnivan Any chance to release a bugfix version, containing this fix?

**Comment by pascalberger** [comment]
Closing since it is already fixed but not released yet.
